README.md

```diff
@@ -20,7 +20,7 @@ This `README` is also available in notebook form:
 
 ```julia
 using Pkg
-Pkd.add("https://github.com/maltezfaria/DataFlowTasks.jl.git")
+Pkg.add(url = "https://github.com/maltezfaria/DataFlowTasks.jl.git")
```

Comment:
Thanks for spotting/fixing this! Since the package is registered, I think 

```julia
Pkg.add("DataFlowTasks")
```

is a better option now.

Reply:
I didn't know that ! Great :smiley: